SparkPost/2web2ui
# Prototype signals details

### What Changed
- Adds a single page prototype 
  - Path: `/prototype/signals-details`
  - UI flag: `prototype_signals_details`
  - Page component is async loaded to avoid bloating the main bundle
- Adds one prop to signals' `BarChart` - `cartesianGridProps`

### How To Test
- Enable the UI option
- Navigate to the route
- Additionally
  - Verify the route is not accessible without the flag
  - Verify the live signals barcharts are unaffected


### To Do
- [ ] Any feedback
